Sources/Plasma/NucleusLib/pnInputCore/plInputMap.cpp

```diff
@@ -704,393 +688,380 @@ void plKeyMap::ICheckAndBindDupe(plKeyDef origKey, plKeyDef dupeKey)
     }   
 }
 
-Win32keyConvert plKeyMap::fKeyConversionEnglish[] =
+const std::map<uint32_t, ST::string> plKeyMap::fKeyConversionEnglish =
 { 
 #ifdef HS_BUILD_FOR_WIN32
-    { VK_F1,    "F1"}, 
-    { VK_F2,    "F2"}, 
-    { VK_F3,    "F3"}, 
-    { VK_F4,    "F4"},
-    { VK_F5,    "F5"},
-    { VK_F6,    "F6"},
-    { VK_F7,    "F7"}, 
-    { VK_F8,    "F8"},
-    { VK_F9,    "F9"},
-    { VK_F10,   "F10"},
-    { VK_F11,   "F11"},
-    { VK_F12,   "F12"},
-    { VK_ESCAPE, "Esc"},
-    { VK_TAB,   "Tab"},
-    { VK_UP,    "UpArrow"}, 
-    { VK_DOWN,  "DownArrow"}, 
-    { VK_LEFT,  "LeftArrow"},
-    { VK_RIGHT, "RightArrow"},
-    { VK_BACK,  "Backspace"},
-    { VK_RETURN, "Enter"}, 
-    { VK_PAUSE, "Pause"},
-    { VK_CAPITAL, "CapsLock"},
-    { VK_PRIOR, "PageUp"},
-    { VK_NEXT,  "PageDn"},
-    { VK_END,   "End"},
-    { VK_HOME,  "Home"},
-    { VK_SNAPSHOT,  "PrintScrn"},
-    { VK_INSERT,    "Insert"},
-    { VK_DELETE,    "Delete"},
-    { VK_NUMPAD0,   "NumPad0"}, 
-    { VK_NUMPAD1,   "NumPad1"}, 
-    { VK_NUMPAD2,   "NumPad2"}, 
-    { VK_NUMPAD3,   "NumPad3"},
-    { VK_NUMPAD4,   "NumPad4"},
-    { VK_NUMPAD5,   "NumPad5"},
-    { VK_NUMPAD6,   "NumPad6"}, 
-    { VK_NUMPAD7,   "NumPad7"},
-    { VK_NUMPAD8,   "NumPad8"},
-    { VK_NUMPAD9,   "NumPad9"},
-    { VK_MULTIPLY,  "NumPad*"},
-    { VK_ADD,       "NumPad+"},
-    { VK_SUBTRACT,  "NumPad-"},
-    { VK_DECIMAL,   "NumPad."},
-    { VK_DIVIDE,    "NumPad/"},
-    { VK_SPACE,     "SpaceBar"},
-    { VK_OEM_COMMA, "Comma"},
-    { VK_OEM_PERIOD,"Period"},
-    { VK_OEM_MINUS, "Minus"},
-    { VK_OEM_PLUS,  "Plus"},
-    { VK_SHIFT,     "Shift" },
+    {VK_F1,    ST_LITERAL("F1")},
```

Comment:
Sorry for missing this earlier, but we [prefer to see spaces](https://github.com/H-uru/Plasma/wiki/Preferred-Code-Style#spacing) used in initializer lists.